OpenOversight/tests/conftest.py

```diff
@@ -48,6 +49,10 @@
     "Chief",
 ]
 
+DEPARTMENT_NAME = "Springfield Police Department"
+DEPARTMENT_TWO_NAME = "Chicago Police Department"
+DEPARTMENT_EMPTY_NAME = "Empty Police Department"
```

Comment:
Since these are paired to specific short names, I think it may be valuable to put them in a dictionary.

```python
DEPARTMENT_KEY_SPD = "SPD"
DEPARTMENT_KEY_CPD = "CPD"
DEPARTMENT_KEY_EPD = "EPD"

DEPARTMENTS = {
    DEPARTMENT_KEY_SPD: "Springfield Police Department",
    DEPARTMENT_KEY_CPD: "Empty Police Department",
    DEPARTMENT_KEY_EPD: "Chicago Police Department"
}
```

You can use the key as the short name (lower or upper case, whateva) and the value as the long name.

Reply:
On second thought, tuples would work as well.
```python
DEPARTMENT_SPD = ("SPD", "Springfield Police Department")
DEPARTMENT_CPD = ("CPD", "Chicago Police Department")
DEPARTMENT_EPD = ("EPD", "Empty Police Department")
```

Reply:
The department short name does not seem to matter much for tests (they are not used anywhere other than the initial creation) so I decided against giving each their own variable.